modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/SetProcessor.java

```diff
@@ -110,10 +111,11 @@ public SetProcessor create(Map<String, Processor.Factory> registry, String proce
                                    String description, Map<String, Object> config) throws Exception {
             String field = ConfigurationUtils.readStringProperty(TYPE, processorTag, config, "field");
             String copyFrom = ConfigurationUtils.readOptionalStringProperty(TYPE, processorTag, config, "copy_from");
+            String mimeType = ConfigurationUtils.readMimeTypeProperty(TYPE, processorTag, config, "mime_type", "application/json");
```

Comment:
are we sure we want `mime_type` as a name here? I thought mime was replace with `media_type` 
but wouldn't `content_type`  be more consistent with what we have in mustache templating engine?

Reply:
or we could consider to rename the content_type to media_type?

Reply:
I'm happy to rename to whatever is most appropriate. I went with `mime_type` because that is what was used in `CustomMustacheFactory` but I would agree that `media_type` or possibly `content_type` would be better.

Reply:
+1 to follow up with a change to `media_type` to be more correct.